diff --git i/visualizer.lua w/visualizer.lua
index 140fa5c..3478aeb 100644
--- i/visualizer.lua
+++ w/visualizer.lua
@@ -7,7 +7,7 @@ local opts = {
     -- novideo          enable visualization when no video
     -- force            always enable visualization
 
-    name = "showcqt",
+    name = "showwaves",
     -- off
     -- showcqt
     -- avectorscope
@@ -32,7 +32,7 @@ local opts = {
 
 -- key bindings
 -- cycle visualizer
-local cycle_key = "c"
+local cycle_key = "C"
 
 if not (mp.get_property("options/lavfi-complex", "") == "") then
     return
@@ -40,10 +40,7 @@ end
 
 local visualizer_name_list = {
     "off",
-    "showcqt",
     "avectorscope",
-    "showspectrum",
-    "showcqtbar",
     "showwaves",
 }
 
