diff --git i/navigator.lua w/navigator.lua
index b7172f2..06a7669 100644
--- i/navigator.lua
+++ w/navigator.lua
@@ -31,9 +31,9 @@ local settings = {
   --favorites in format { 'Path to directory, notice trailing /' }
   --on windows use double backslash c:\\my\\directory\\
   favorites = {
-    '/media/HDD2/music/music/',
-    '/media/HDD/users/anon/Downloads/',
-    '/home/anon/',
+    -- '/media/HDD2/music/music/',
+    -- '/media/HDD/users/anon/Downloads/',
+    -- '/home/anon/',
   },
   --list of paths to ignore. the value is anything that returns true for if-statement.
   --directory ignore entries must end with a trailing slash,
