diff --git i/M-x.lua w/M-x.lua
index 2decf99..79f3d6f 100644
--- i/M-x.lua
+++ w/M-x.lua
@@ -9,7 +9,7 @@ local opts = {
   sort_commands_by = 'priority',
 
   -- options for extended menu ------------------------------------------------
-  toggle_menu_binding = 't',
+  toggle_menu_binding = 'Alt+x',
   lines_to_show = 17,
   pause_on_open = true,
   resume_on_exit = "only-if-was-paused", -- another possible value is true
@@ -38,9 +38,7 @@ local opts = {
 
 (require 'mp.options').read_options(opts, mp.get_script_name())
 
-package.path =
-    mp.command_native({ "expand-path", "~~/script-modules/?.lua;" }) .. package.path
-local em = require "extended-menu"
+local em = dofile("extended-menu")
 
 local mx_menu = em:new(opts)
 
